.pre-commit-config.yaml

```diff
@@ -44,6 +45,22 @@ repos:
           - stylelint-config-standard@32.0.0
           - stylelint-prettier@3.0.0
 
+  - repo: local
```

Comment:
We have these hooks but we kept `eslint` and `stylelint` ?
Also let's keep in mind using `task` for linting does not handle per-file linting during pre-commit, so it might get beefy at one point.

I'm not against **not using** task within pre-commits.

Also, we mentioned using [biome](https://biomejs.dev/) at one point ? it might simplify the whole linting/prettifying process a lot (not for this PR though)

Reply:
Forgot to answer but thanks you're right, I removed `eslint` and `stylelint`, and now pre-commit directly calls `npx sgmf-scripts --lint` instead of the task commands
For biome, yes indeed we mentioned it, but yes it should be handled in another PR (And I guess, in a dedicated project to use biome for all front projects😅)